### ISSUE TITLE
bsnes/heuristics/game-boy.cpp — 3 lines after `if` are indented, but not in block

### ISSUE THREAD

**Issue by Marat-Tanalin** [bsnes-emu/bsnes#281]
Found suspicious code in [bsnes/heuristics/game-boy.cpp#L277](https://github.com/bsnes-emu/bsnes/blob/5cefce5c08f74cfc80eee3f82a32d846144e5277/bsnes/heuristics/game-boy.cpp#L277) that contains 3 lines indented, but not wrapped in a `{}` block, so only the first of them effectively depends on the condition:

```
if(serial)
  output.append("  serial: ", serial, "\n");
  output.append("  board:  ", mapper, "\n");
  output.append(Memory{}.type("ROM").size(data.size()).content("Program").text());
```

Not sure what this code does, but this might be a bug.

**Comment by jchv** [comment]
Looks like it's not a bug, just weird code formatting, probably to keep the statements aligned. Notice that only the statement covered by the conditional pertains to it.

**Comment by Marat-Tanalin** [comment]
Yeah, the first line just explicitly contains the variable from the condition, but the other two lines might make sense depending on that variable too.

**Comment by orbea** [comment]
The others are always present, only `serial` might be empty.
